recipes/gw/fix-linker-issues.patch

```diff
@@ -0,0 +1,5 @@
+--- a/Makefile
++++ b/Makefile
+@@ -132,1 +132,1 @@
+-	$(CXX) $(CXXFLAGS) $(OBJECTS) $(LDFLAGS) $(LDLIBS) -o $@
++	$(CXX) $(CXXFLAGS) $(OBJECTS) $(LDFLAGS) $(LDLIBS) -Wl,-rpath,$(PREFIX)/lib -Wl,-rpath-link,$(PREFIX)/lib -lxcb-glx -lxcb-dri2 -lexpat -o $@
```

Comment:
Again, I think there is no need of this patch. You could pass `-Wl,-rpath,$(PREFIX)/lib -Wl,-rpath-link,$(PREFIX)/lib` with `LDFLAGS` and `-lxcb-glx -lxcb-dri2 -lexpat` with `LDLIBS` in build.sh

Reply:
Hi @martin-g,
I see what you mean now